player/lua/select.lua

```diff
@@ -353,6 +353,106 @@ mp.add_key_binding(nil, "select-audio-device", function ()
     })
 end)
 
+local function get_history_entry(line, seen, error_message)
+    local entry = utils.parse_json(line)
+
+    if not entry then
+        mp.msg.warn(error_message)
+        return
+    end
+
+    local time, path, title = unpack(entry)
+
+    if not path then
+        mp.msg.warn(error_message)
+        return
+    end
+
+    if seen[path] then
+        return
+    end
+
+    seen[path] = true
+
+    local status, date = pcall(os.date, "(%Y-%m-%d %H:%M) ", time)
```

Comment:
This date format should be configurable and should include seconds by default.

Reply:
Added a script-opt but there's nowhere to document it until I add a select.lua section to the docs.

Reply:
Also added `hide_history_duplicates` script-opt that's true by default since I prefer to show them.